docs/reference/data-tables.md

```diff
@@ -182,3 +182,154 @@ numbers, filesizes, dates and month names. See the [tablesort documentation]
 
   [tablesort]: http://tristen.ca/tablesort/demo/
   [instant loading]: ../setup/setting-up-navigation.md#instant-loading
+
+### Import table from file
+
+[:octicons-cpu-24: Plugin][table-reader-docs]
+
+You can also import data from a CSV or Excel file using the plugin [`mkdocs-table-reader-plugin`][table-reader-docs].
+
+First, you will need to install it with `pip`:
+
+```sh
+pip install mkdocs-table-reader-plugin
+```
+
+Then extend the `mkdocs.yml` file like this:
+
+```yaml
+plugins:
+  - table-reader
+```
+
+Then, you can import the data like this:
+
+#### Example 1: Import data from CSV file :fontawesome-solid-file-csv:
+    
+Let's use a :fontawesome-solid-file-csv: CSV in the local directory.
+
+=== "Data :fontawesome-solid-file-csv:"
+    
+    The :fontawesome-solid-file-csv: CSV file may look like this:
+
+    ```csv title="./data.csv"
+    col1,col2,col3
+    r1c1,r1c2,r1c3
+    r2c1,r2c2,r2c3
+    r3c1,r3c2,r3c3
+    ```
+
+=== "Markdown :fontawesome-solid-file-arrow-down:"
+
+    You can add it to your :fontawesome-solid-file-arrow-down: Markdown page like this:
+
+    ```md title="./markdown.md"
+    ...
+    {{ read_csv('./data.csv') }}
+    ...
+    ```
+
+And it will then return a result like this:
+
+{{ read_csv('./data.csv') }}
```

Comment:
You didn't add the plugin to `mkdocs.yml`, so this will be printed as-is (= no table). However, I would prefer if we don't add it to `mkdocs.yml`, but it means we need to include the plugin output manually, so please do so. We do the same at different positions of our documentation, because sometimes options are even conflicting, so we have to "fake" it. Additionally, we need to try and keep the dependencies minimal.

Additionally, please remove both data files.

Reply:
When including the plugin output, please use Markdown table syntax.